src/components/ListedSpecies/Location.jsx

```diff
@@ -0,0 +1,90 @@
+import { InfoIcon } from '@chakra-ui/icons';
+import {
+  Flex,
+  FormControl,
+  FormLabel,
+  Grid,
+  GridItem,
+  HStack,
+  Input,
+  Text,
+  Tooltip,
+} from '@chakra-ui/react';
+import PropTypes from 'prop-types';
+import { IMaskInput, useIMask } from 'react-imask';
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'useIMask' is defined but never used.